test/core/security/security_connector_test.cc

```diff
@@ -28,6 +28,7 @@
 #include "src/core/lib/gpr/string.h"
 #include "src/core/lib/gpr/tmpfile.h"
 #include "src/core/lib/security/context/security_context.h"
+#include "src/core/lib/security/security_connector/security_connector.cc"
```

Comment:
Why you need this?

Reply:
`ssl_host_matches_name` is `static`. Would love ideas on how to make this better.

Reply:
see: https://github.com/grpc/grpc/pull/14387#discussion_r171817241

Reply:
Please don't include .cc file in the header. 

You can make ssl_host_matches_name() a non-static function, and add the function declaration at security_connector.h, on the section of "exposed for testing only".
https://github.com/grpc/grpc/blob/master/src/core/lib/security/security_connector/security_connector.h#L245

In the future, security_connector will be re-written in C++, we will have better way of testing through mock testing class.

Reply:
No problem. Sounds good.